community/kernel/src/test/java/org/neo4j/kernel/impl/core/TestRelationship.java

```diff
@@ -557,6 +559,16 @@ public void testRelGetProperties()
         assertTrue( rel1.hasProperty( key1 ) );
         assertTrue( rel1.hasProperty( key2 ) );
         assertTrue( rel1.hasProperty( key3 ) );
+
+        Map<String, Object> properties = rel1.getAllProperties();
+        assertTrue( properties.get( key1 ).equals( int1 ) );
+        assertTrue( properties.get( key2 ).equals( int2 ) );
+        assertTrue( properties.get( key3 ).equals( string ) );
+        properties = rel1.getProperties( key1, key2 );
+        assertTrue( properties.get( key1 ).equals( int1 ) );
+        assertTrue( properties.get( key2 ).equals( int2 ) );
+        assertFalse( properties.containsKey( key3 ) );
+
```

Comment:
Can we please split this method in several ones with proper test-explanatory names?